### PR TITLE
DOT-4690: We changed values in Order_Action to integer

### PR DESCRIPTION
**Pull request by olyans**

### CODE REVIEW

File: includes/class-wc-gateway-youcanpay.php [hunk 1]
```diff
@@ -188,15 +188,12 @@ public function javascript_params() {
...
-			'is_pre_order'     => WC_YouCanPay_Order_Action_Enum::get_incomplete(),
-			'order_status'     => array(
-				'incomplete' => WC_YouCanPay_Order_Action_Enum::get_incomplete(),
-				'pre_order'  => WC_YouCanPay_Order_Action_Enum::get_pre_order(),
-			),
+			'gateway'          => self::ID,
+			'locale'           => WC_YouCanPay_Helper::get_supported_local( get_locale() ),
+			'checkout_url'     => WC_YouCanPay_Helper::get_ajax_checkout_url(),
+            'is_test_mode'     => $this->is_in_test_mode(),
+			'is_pre_order'     => 0,
```
Comment (AmineAlyate) [inline includes/class-wc-gateway-youcanpay.php:195]: indent the array

File: includes/enums/class-wc-youcanpay-order-action-action-enum.php [hunk 2]
```diff
@@ -23,7 +27,11 @@ public static function get_pre_order(): string {
 		return self::$pre_order;
 	}
 
-	public static function get_all() {
+    /**
+     * @return array
+     */
+	public static function get_all(): array
```
Comment (AmineAlyate) [inline includes/enums/class-wc-youcanpay-order-action-action-enum.php:33]: space 🙏

File: includes/class-wc-gateway-youcanpay.php (hunk 1 above)
Comment (olyans) [reply includes/class-wc-gateway-youcanpay.php]: https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/11/commits/bed2f80778af4957fea82ef435ba424548b56d3a

File: includes/enums/class-wc-youcanpay-order-action-action-enum.php (hunk 2 above)
Comment (olyans) [reply includes/enums/class-wc-youcanpay-order-action-action-enum.php]: https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/11/commits/bed2f80778af4957fea82ef435ba424548b56d3a

**Review by AmineAlyate — APPROVED** [review]
LGTM :heavy_check_mark:

**Pull request merged**